easybuild/easyconfigs/c/cairo/cairo-1.12.18-goolf-1.7.20.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = "cairo"
+version = '1.12.18'
+
+homepage = 'http://cairographics.org'
+description = """Cairo is a 2D graphics library with support for multiple output devices.
+Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
+PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB"""
```

Comment:
if you can do 1 leading space here, that's nice
(sry to nag about it, yet I've been reviewed in the same way!).